data/wanderers middle.txt

```diff
@@ -2083,11 +2087,10 @@ mission "Wanderers: Sestor: Bomb Zenith 1"
 	to offer
 		has "Wanderers: Sestor: Quarg Help 2: done"
 	cargo "unknown device" 2
-	blocked "You will need two tons of cargo space to take on the next mission for Danforth."
 	on offer
-		log "Recruited some Quarg warships to help defend human space from the Kor Sestor drones, which are now presumably under Alpha control. Danforth believes that he has located the Alpha base from which the drones are being controlled; if the base is destroyed the drones ought to become aimless and be easily defeated."
```

Comment:
Why this change?

Reply:
>Splits the log entry for the Quarg escorts (as some players choose instead to wander with them, and may forget exactly when they picked up these escorts).

The log bit about recruiting Quarg escorts is now journaled when you recruit them, as opposed to when you deliver them